.github/workflows/test-fedora-binaries.yml

```diff
@@ -0,0 +1,19 @@
+name: Test Fedora binaries (Sequential)
```

Comment:
Do you think it's a good idea to split tests in different workflows? Semantically, fedora and ubuntu tests seem to be part of the one workflow.